### PR TITLE
Improved: dropdown close area

### PR DESCRIPTION
**Pull request by math-GH**
Changes proposed in this pull request:

- some HTML improvements: every dropdown menu has the same "close button" structure like the mobile aside bar (f.e. in the normal view). It means: `.dropdown-close` is not anymore a part of the `.dropdown-menu`, but it is the next element.
- CSS is cleaned up
- visible new result: grey clickable area left of the mobile config menu
grafik

How to test the feature manually:

1. use FreshRSS on a small display ("mobile view")
2. open the config dropdown menu
3. see the grey area left of the menu
4. click on it
5. the menu closes

Plus: Check all dropdown menus:
normal view:
- user queries dropdown (top menu)
- mark as read dropdown (top menu)
- share dropdown (below the article)
- tags dropdown (below the article)
- labels dropdown (below the article)
- feed dropdown (left navigation)

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
It is the same greyed background like in #4292 and #4061

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -1769,7 +1756,7 @@ input:checked + .slide-container .properties {
 		bottom: 0;
 		left: 0;
 		right: 0;
-		cursor: pointer;
+		cursor: default;
```
Comment (Frenzie) [inline p/themes/base-theme/template.css:1759]: Is there somewhere else in the cascade that changes it? As written it looks like the line should simply be removed.

File: p/themes/base-theme/template.css [hunk 2]
```diff
@@ -527,32 +527,18 @@ input[type="checkbox"]:focus-visible {
...
-
-.dropdown-target:target ~ div.dropdown-close {
-	display: block;
 	z-index: 999;
-	position: relative;
-}
-
-.dropdown-menu-scrollable .dropdown-close {
-	display: none;
+	cursor: default;
```
Comment (Frenzie) [inline p/themes/base-theme/template.css:541]: Same here.

File: p/themes/base-theme/template.css (hunk 1 above)
Comment (math-GH) [reply p/themes/base-theme/template.css:1759]: `.dropdown-close` is a `<a>` and the default cursor would be the "hand" icon.
`cursor: default` = maus pointer (arrow).
Maybe you mix it with `cursor: auto`?

grafik

Comment (Frenzie) [reply p/themes/base-theme/template.css:1759]: No, I was just commenting on the diff.

Comment (math-GH) [reply p/themes/base-theme/template.css:1759]: I checked. It is correct as it is

Comment (Alkarex) [reply p/themes/base-theme/template.css:1759]: I did not follow the whole discussion. What is the status? :-)

Comment (Frenzie) [reply p/themes/base-theme/template.css:1759]: All good probably.

**Pull request merged**